tecken/download/views.py

```diff
@@ -36,64 +37,73 @@
 )
 
 
-def _ignore_symbol(symbol, debugid, filename):
+def _ignore_symbol(debugfilename, debugid, filename):
     # The MS debugger will always try to look up these files. We
     # never have them in our symbol stores. So it can be safely ignored.
     if filename == "file.ptr":
         return True
 
+    if debugid == BOGUS_DEBUG_ID:
+        return True
```

Comment:
In the previous iteration of this change, we were using the bogus debug id as an indication that we should do a lookup using the `code_file` and `code_id` for the real debug id. Because of that, we needed to not ignore those requests. However, in the case where the debug id is bogus, the debug filename is also missing and the stackwalker won't attempt to use the download API to get the symbol, so that iteration of the change would never work.

Now that we're changing how we're supporting `code_file` / `code_id` lookups, I re-added this.